### PR TITLE
Display only current site elements in dashboard.

### PR DESCRIPTION
**Pull request by itkg-canne**
Depends on https://github.com/open-orchestra/open-orchestra-model-bundle/pull/468.

### CODE REVIEW

File: ApiBundle/Controller/ContentController.php [hunk 1]
```diff
@@ -263,18 +263,24 @@ protected function findOneContent($contentId, $language, $version = null)
     /**
      * @param boolean|null $published
      *
-     * @Config\Route("/list/not-published-by-author", name="open_orchestra_api_content_list_author_not_published", defaults={"published": false})
-     * @Config\Route("/list/by-author", name="open_orchestra_api_content_list_author", defaults={"published": null})
+     * @Config\Route("/list/not-published-by-author", name="open_orchestra_api_content_list_author_and_site_not_published", defaults={"published": false})
```
Comment (itkg-ngilain) [inline ApiBundle/Controller/ContentController.php]: I'm not sure about the route name which is becoming truly heavy. Maybe you could rename it into something more straight, and maybe something speaking of dashbord widget as this controller exists only for this functionnality.

File: ApiBundle/Controller/ContentController.php [hunk 2]
```diff
@@ -263,18 +263,24 @@ protected function findOneContent($contentId, $language, $version = null)
...
     {
+        $siteId = $this->get('open_orchestra_backoffice.context_manager')->getCurrentSiteId();
         $user = $this->get('security.token_storage')->getToken()->getUser();
-        $content = $this->get('open_orchestra_model.repository.content')->findByAuthor($user->getUsername(), $published, 10);
+        $content = $this->get('open_orchestra_model.repository.content')->findByAuthorAndSiteId(
+            $user->getUsername(),
+            $siteId,
+            $published,
+            10
+            );
```
Comment (itkg-ngilain) [inline ApiBundle/Controller/ContentController.php]: bad ident

File: ApiBundle/Controller/NodeController.php [hunk 3]
```diff
@@ -134,18 +134,24 @@ public function duplicateAction(Request $request, $nodeId, $version = null)
     /**
      * @param boolean|null $published
      *
-     * @Config\Route("/list/not-published-by-author", name="open_orchestra_api_node_list_author_not_published", defaults={"published": false})
-     * @Config\Route("/list/by-author", name="open_orchestra_api_node_list_author", defaults={"published": null})
+     * @Config\Route("/list/not-published-by-author", name="open_orchestra_api_node_list_author_and_site_not_published", defaults={"published": false})
```
Comment (itkg-ngilain) [inline ApiBundle/Controller/NodeController.php]: same note here

File: ApiBundle/Controller/NodeController.php [hunk 4]
```diff
@@ -134,18 +134,24 @@ public function duplicateAction(Request $request, $nodeId, $version = null)
...
     {
+        $siteId = $this->get('open_orchestra_backoffice.context_manager')->getCurrentSiteId();
         $user = $this->get('security.token_storage')->getToken()->getUser();
-        $nodes = $this->get('open_orchestra_model.repository.node')->findByAuthor($user->getUsername(), $published, 10);
+        $nodes = $this->get('open_orchestra_model.repository.node')->findByAuthorAndSiteId(
+            $user->getUsername(),
+            $siteId,
+            $published,
+            10
+            );
```
Comment (itkg-ngilain) [inline ApiBundle/Controller/NodeController.php]: bad ident too

File: ApiBundle/Controller/ContentController.php (hunk 1 above)
Comment (itkg-canne) [inline ApiBundle/Controller/ContentController.php]: I think API methods should be generic, so I not agree with giving to routes a name related to a specific view.

File: ApiBundle/Controller/NodeController.php (hunk 3 above)
Comment (itkg-canne) [inline ApiBundle/Controller/NodeController.php]: same note here

File: ApiBundle/Controller/NodeController.php (hunk 4 above)
Comment (itkg-canne) [inline ApiBundle/Controller/NodeController.php]: Fixed.

File: ApiBundle/Controller/ContentController.php (hunk 2 above)
Comment (itkg-canne) [inline ApiBundle/Controller/ContentController.php]: Fixed.

**Comment by itkg-canne** [comment]
@itkg-ngilain 
up

**Comment by itkg-nanne** [comment]
:+1:

**Comment by ggrasset** [comment]
:+1:

**Pull request merged**